src/addons/transitions/__tests__/ReactTransitionGroup-test.js

```diff
@@ -94,15 +92,15 @@ describe('ReactTransitionGroup', function() {
 
   it('should handle enter/leave/enter/leave correctly', function() {
     var log = [];
-    var cb;
+    var willEnterCb;
```

Comment:
`componentWillLeave` also defined `cb` so there was shadowing. Additionally, the existence of `_cb` and `cb` seemed a bit ugly. This seemed like the smallest modification I could make that didn't sacrifice readability.